system-tests/projects/no-specs-vue/cypress/support/component.ts

```diff
@@ -0,0 +1,43 @@
+// ***********************************************************
+// This example support/component.ts is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```

Comment:
Delete?

Reply:
the whole support file or just this line? this line just gets generated in scaffolding cypress is the only reason I ask because we don't use this support file for anything

Reply:
Oh ok. Up to you then. I just noticed it was commented out 🤷🏻